website/docs/guides/athena-qs.md

```diff
@@ -0,0 +1,334 @@
+- You have an [AWS account](https://aws.amazon.com/).
+- You have set up [Amazon Athena](https://docs.aws.amazon.com/athena/latest/ug/getting-started.html).
+
+### Related content
+
+- Learn more with [dbt Learn courses](https://learn.getdbt.com)
+- [CI jobs](/docs/deploy/continuous-integration)
+- [Deploy jobs](/docs/deploy/deploy-jobs)
+- [Job notifications](/docs/deploy/job-notifications)
+- [Source freshness](/docs/deploy/source-freshness)
+
+## Getting started
+
+For the following guide you can use an existing S3 bucket or [create a new one](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html).
+
+Download the following CSV files (the Jaffle Shop sample data) and upload them to your S3 bucket:
+- [jaffle_shop_customers.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_customers.csv)
+- [jaffle_shop_orders.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/jaffle_shop_orders.csv)
+- [stripe_payments.csv](https://dbt-tutorial-public.s3-us-west-2.amazonaws.com/stripe_payments.csv)
+
+
+## Configure Amazon Athena
+
+1. Log into your AWS account and navigate to the **Athena console**.
+    - If this is your first time in the Athena console (in your current AWS Region), click **Explore the query editor** to open the query editor. Otherwise, Athena opens automatically in the query editor.
+1. Open **Settings** and find the **Location of query result box** field.
+    1. Enter the path of the S3 bucket (prefix it with `s3://`).
+    2. Navigate to **Browse S3**, select the S3 bucket you created, and click **Choose**.
+1. **Save** these settings.
+1. In the **query editor**, create a database by running `create database YOUR_DATABASE_NAME`.
+1. To make the database you created the one you `write` into, select it from the **Database** list on the left side menu. 
+1. Access the Jaffle Shop data in the S3 bucket using one of these options:
+    1. Manually create the tables.
+    2. Create a glue crawler to recreate the data as external tables (recommended).
+1. Once the tables have been created, you will able to `SELECT` from them. 
+
+## Set up security access to Athena
+
+To setup the security access for Athena, determine which access method you want to use: 
+* Obtain `aws_access_key_id` and `aws_secret_access_key` (recommended)
+* Obtain an **AWS credentials** file.
+
+### AWS access key (recommended)
+
+To obtain your `aws_access_key_id` and `aws_secret_access_key`:
+
+1. Open the **AWS Console**.
+1. Click on your **username** near the top right and click **Security Credentials**.
+1. Click on **Users** in the sidebar.
+1. Click on your **username** (or the name of the user for whom to create the key).
+1. Click on the **Security Credentials** tab.
+1. Click **Create Access Key**.
+1. Click **Show User Security Credentials** and 
+
+Save the `aws_access_key_id` and `aws_secret_access_key` for a future step.
+
+### AWS credentials file
```

Comment:
@amychen1776 Should we even include this?

Reply:
I think we should keep this!